src/test/resources/restapi/roles_null_array_element.json

```diff
@@ -0,0 +1,9 @@
+{
+  "cluster_permissions" : [null],
+  "index_permissions" : [ {
+    "index_patterns" : [ "*" ],
```

Comment:
Add use case for `null` in "index_patterns", "masked_fields" and "allowed_actions" and any of those being `null` by itself.

Reply:
Done